entproto/cmd/protoc-gen-entgrpc/converter.go

```diff
@@ -65,7 +66,21 @@ func (g *serviceGenerator) newConverter(fld *entproto.FieldMappingDescriptor) (*
 	if fld.IsEdgeField {
 		efld = fld.EntEdge.Type.ID
 	}
+
 	switch {
+	case efld.IsUUID() && efld.Type.ValueScanner():
+		out.toEntScannerConstructor = protogen.GoImportPath("github.com/google/uuid").Ident("UUID")
+	case efld.HasGoType() && efld.Type.ValueScanner():
+		// Ident returned from ent already has the packagename prefixed. Strip it since `g.QualifiedGoIdent`
+		// adds it back.
```

Comment:
@rotemtam @a8m please let me know if there is a cleaner way to do this

Reply:
I think that `protogen.GoImportPath(split[1]).Ident(efld.Type.PkgPath)` should be ok?